app/assets/javascripts/lara-api.js

```diff
@@ -140,15 +143,33 @@ window.LARA = {
       remove: remove
     }
   },
+  /****************************************************************************
+  @deprecated saveLearnerState
+  @see saveLearnerPluginState
+  ****************************************************************************/
+  saveLearnerState: function (pluginId, state) {
+    var deprication =
+      '⚠️ saveLearnerState is depricated, use `saveLearnerPluginState` instead'
+    console.warn(deprication);
+    this.saveLearnerPluginState(pluginId, state);
+  },
 
   /****************************************************************************
-   @function saveUserState: Ask LARA to save the users state for the plugin
-   @arg {ILaraPluginRef} pluginInstance - The plugin trying to save data
+   @function saveLearnerPluginState: Ask LARA to save the users state for the plugin
```

Comment:
Nice, now I get what was the intention. saveLearnerState wasn't really related to the plugin. This seems perfect, thanks.